core-java/src/main/java/com/baeldung/utils/Utils.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.utils;
```

Comment:
Let's put the code in the matching package structure, in this case, we could use "com.baeldung.chartoarray"